documentation/modules/ROOT/pages/operations/openshift.adoc

```diff
@@ -8,249 +8,335 @@
+        - name: MYSQL_USER
+          value: mysqluser
+        - name: MYSQL_PASSWORD
+          value: mysqlpw
+        ports:
+        - containerPort: 3306
+          name: mysql
+EOF
 ----
-+
-After a while all parts should be up and running:
-+
-[source%nowrap,bash]
+
+== Deploying a {prodname} Connector
+
+To deploy a {prodname} connector, you need to deploy a Kafka Connect cluster with the required connector plug-in(s), before instantiating the actual connector itself.
+As the first step, a container image for Kafka Connect with the plug-in has to be created.
+If you  already have a container image built and available in the registry, you can skip this step.
+In this document, the MySQL connector will be used as an example.
+
+=== Creating Kafka Connect Cluster
+
+Again, we will use Strimzi for creating the Kafka Connect cluster.
+Strimzi also can be used for building and pushing the required container image for us.
+In fact, both tasks can be merged together and instructions for building the container image can be provided directly within the `KafkaConnect` object specification:
+
+[source,bash]
 ----
-oc get pods
-
-NAME                                       READY     STATUS        RESTARTS   AGE
-broker-entity-operator-5fb7bc8b9b-r86nz    3/3       Running       1          4m
-broker-kafka-0                             2/2       Running       0          4m
-broker-zookeeper-0                         2/2       Running       0          5m
-debezium-connect-3-4sdjr                   1/1       Running       0          1m
-strimzi-cluster-operator-d77476b8f-rblqf   1/1       Running       0          5m
+$ cat << EOF | oc create -f -
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: debezium-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+spec:
+  version: 3.1.0
+  replicas: 1
+  bootstrapServers: debezium-cluster-kafka-bootstrap:9092
+  config:
+    group.id: connect-cluster
+    offset.storage.topic: connect-cluster-offsets
+    config.storage.topic: connect-cluster-configs
+    status.storage.topic: connect-cluster-status
+    # -1 means it will use the default replication factor configured in the broker
+    config.storage.replication.factor: -1
+    offset.storage.replication.factor: -1
+    status.storage.replication.factor: -1
+  build:
+    output:
+      type: docker
+      image: image-registry.openshift-image-registry.svc:5000/debezium-example/debezium-connect-mysql:latest
+    plugins:
+      - name: debezium-mysql-connector
+        artifacts:
+          - type: tgz
+            url: https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/{debezium-version}/debezium-connector-mysql-{debezium-version}-plugin.tar.gz
+EOF
 ----
-+
-Alternatively, go to the "Pods" view of your OpenShift Web Console (https://myhost:8443/console/project/myproject/browse/pods) to confirm that all pods are up and running:
-+
-image::openshift_pods.png[width=771,align="center"]
```

Comment:
Any chance to replace it with a current one instead of removing it? Our docs are very text-heavy in general, so anything visual is always appreciated :)

Reply:
ack